contracts/delegate.md

```diff
@@ -144,6 +149,12 @@ function setRuleAndIntent(
 A successful `setRuleAndIntent` will emit a `SetRule` event and `Stake` event. It will be an
 all-or-nothing transaction.
 
+| Revert Reason               | Scenario                                                       |
+| :-------------------------- | :------------------------------------------------------------- |
+| `INVALID_PRICE_COEF`     | The priceCoef must be greater than 0.                 |
+| `STAKING_TRANSFER_FAILED`     | The Delegate contract was not approved to transfer the staking token to itself.                 |
+| `STAKING_RETURN_FAILED`     | The Delegate was unable to transfer remaining staked amount back. Please increase gas limit.                 |
```

Comment:
I'd remove `Please increase gas limit.` it feels like an excess statement. Gas limit isnt the only reason token transfers can fail.